lib/main.dart

```diff
@@ -1,24 +1,59 @@
 import 'package:flutter/material.dart';
+import 'homepage_options/choose_option.dart';
 
 void main() {
   runApp(const MyApp());
 }
 
 class MyApp extends StatelessWidget {
   const MyApp({Key? key}) : super(key: key);
+  // Moet nog veranderd worden naar ChooseType()
+  void ChooseStudent() {
+    print('Student chosen');
+  }
+
+  void ChooseDocent() {
+    print('Docent chosen');
+  }
 
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
+      title: 'Choose type',
       theme: ThemeData(
-        primarySwatch: Colors.blue,
+        primarySwatch: Colors.red,
       ),
       home: Scaffold(
-        appBar: AppBar(title: const Text('Home')),
-        body: const Text('Hello World!'),
-      ),
+          appBar: AppBar(),
+          body: Column(
+            children: [
+              Container(
+                child: const Text(
+                  "EXAMAP!",
+                  style: TextStyle(
+                    fontSize: 55.0,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+                padding: const EdgeInsets.only(
+                  bottom: 100.0,
+                ),
+              ),
+              Container(
+                child: const Text(
+                  "Gelieve je rol te kiezen",
+                  style: TextStyle(
+                    fontSize: 30.0,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+                padding: EdgeInsets.only(bottom: 100.0),
+              ),
+              ChooseOption(),
+            ],
+            mainAxisAlignment: MainAxisAlignment.center,
+          )),

```

Comment:
```suggestion
          ),),

```
Als je een komma plaatst op lijn 56 en bestand opslaat zal de code automatisch mooi geformateerd zijn ;)